.jpmignore

```diff
@@ -11,3 +11,7 @@ node_modules/
 .stylelintrc
 .travis.yml
 testpilot-containers.xpi
+.vimrc
+.DS_Store
+webextension/.DS_Store
```

Comment:
nit: this is an extra line - the above `.DS_Store` line will ignore all `.DS_Store` files.